hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/BlockingRpcClient.java

```diff
@@ -72,6 +74,14 @@ protected BlockingRpcConnection createConnection(ConnectionId remoteId) throws I
     return new BlockingRpcConnection(this, remoteId);
   }
 
+  @Override
+  public RpcChannel createHedgedRpcChannel(Set<ServerName> sns, User user, int rpcTimeout)
```

Comment:
There's no need for an implementation of this method on this class -- it doesn't support this part of the interface and the parent abstract class's implementation is sufficient.